scripts/CleanCluster.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+if [[ $CLUSTER_TYPE == "PSI" ]]; then
+    ￼ #PSI cluster login
+    ￼ oc login -u kubeadmin -p ${OCP4X_KUBEADMIN_PASSWORD} --insecure-skip-tls-verify ${OCP4X_API_URL}
+else
+    ￼ # Login to IBM Cloud using service account API Key
+    ￼ ##ibmcloud login --apikey $IBMC_OCP47_APIKEY -a cloud.ibm.com -r eu-de -g "Developer CI and QE"
```

Comment:
Why have you commented this? For the time being, it's anyway going to run on PSI, right? When we shift to IBM Cloud, we would have to remember to uncomment this part so that things work on it. Hence, suggesting to not keep it commented.

Reply:
understood